jobs/resource_job.go

```diff
@@ -631,16 +623,125 @@ var jobSchema = common.StructToSchema(JobSettings{},
 			Computed: true,
 		}
 		s["always_running"] = &schema.Schema{
-			Optional: true,
-			Default:  false,
-			Type:     schema.TypeBool,
+			Optional:      true,
+			Default:       false,
+			Type:          schema.TypeBool,
+			Deprecated:    "always_running will be replaced by control_run_state in the next major release.",
+			ConflictsWith: []string{"control_run_state"},
```

Comment:
Good point. I think that would be a breaking change right? Wouldn't we want to save that for a major version change, at which point we'll just remove the field entirely? Today, at least the job is restarted, even if TF provider fails if the job is continuous, but this would block the update in the first place.

Reply:
if it leads to problems I think that it could be ok (we have both github issue & internal ES ticket because of using continuous with always_running)